demos/bazel/src/main/proto/nested.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+// Protos are imported using the path from package root to proto.
+// In the future, specifying a (strip_)import_prefix in the proto_library rule will make this less verbose.
+// For now this blocks on https://github.com/grpc/grpc-java/pull/5621
+import "src/main/proto/helloworld.proto";
```

Comment:
These kinds of proto imports are only needed with Bazel, right?

Reply:
Do you mean the full path from the repository root? Yes (although I don't know how other build systems solve this).
Would you like me to add a line that specifically mentions bazel?

Reply:
Yes, please. The maven and gradle plugins only require relative import paths.

Reply:
Done